src/components/ForecastInfo/ForecastInfo.jsx

```diff
@@ -0,0 +1,47 @@
+import PropTypes from 'prop-types';
+import { SearchBar } from 'components/SearchBar/SearchBar';
+import s from 'components/ForecastInfo/ForecastInfo.module.scss';
+
+export const ForecastInfo = ({ Forecastweather, onChangeSubmit }) => {
+  return (
+    <section className={s.section}>
+      <SearchBar onSubmit={onChangeSubmit} />
+      <h1 className={s.forecastTitle}>{Forecastweather.city.name}</h1>
+
+      <div className={s.forecastInfoThumb}>
+        <img
+          className={s.icon}
+          src={`http://openweathermap.org/img/wn/${Forecastweather.list[0].weather[0].icon}@2x.png`}
+          alt={Forecastweather.list[0].weather[0].icon.description}
+        />
+
+        <p>{Math.round(Forecastweather.list[0].main.temp)}°</p>
+        <p>feels like {Math.round(Forecastweather.list[0].main.feels_like)}°</p>
+        <p> wind speed {Forecastweather.list[0].wind.speed}m/s</p>
```

Comment:
same

Reply:
As far as I understand, if the object with the weather does not come,  error will be in catch. Or the question is if the object will arrive partially? I didn't quite understand the question.

Reply:
Fix